lib/human_player.py

```diff
@@ -1,3 +1,17 @@
 class HumanPlayer:
-    def __init__(self, mark):
+    def __init__(self, ui, board, mark):
+        self.ui = ui
+        self.board = board
         self.mark = mark
+
+    def mark_position(self):
+        self.ui.display("Choose a number between 1 and 9:")
```

Comment:
I realized during your demo that this message is probably not clear. Maybe say instead
> Choose an open space between 1 and 9:

Reply:
Done!